ext/node/polyfills/_fs/_fs_exists.ts

```diff
@@ -16,7 +16,7 @@ type ExistsCallback = (exists: boolean) => void;
  */
 export function exists(path: string | URL, callback: ExistsCallback) {
   path = path instanceof URL ? pathFromURL(path) : path;
-  Deno.lstat(path).then(() => callback(true), () => callback(false));
```

Comment:
should we apply the perf optimizations directly to Deno.lstat?

Reply:
lstat is already optimized but it does some extra stuff that is not needed for exists.